src/StaticAnalysis/ExecutableLinesFindingVisitor.php

```diff
@@ -165,6 +165,14 @@ private function getLines(NodeAbstract $node, bool $fromReturns): array
         }
 
         if ($node instanceof BinaryOp) {
+            if ($node instanceof BinaryOp\Concat &&
```

Comment:
how is concat special vs. other binary operators line plus?

Reply:
I have no idea, but it seems it is special.

I guess it is because it can trigger typecasting on object by a `__toString()` call, wether other binary ops don't

Reply:
I had to revert it, as it is unstable (line is not present in xdebug) when the var expr is known (from constatnt expr).

Reply:
I have to say, I'm kind of disappointed on how you are handling this contribution.

I created a valid test for a valid use case, I got the test passing, you found another edge case and istead of adding them to the tests, you throw my contribution away?

I'm losing my willingess to help, if you discard my help

Reply:
I am sorry to hear that, I already spent about 2 - 3 MDs on static analyser to be really a minimal subset of xdebug and implementing https://github.com/sebastianbergmann/php-code-coverage/issues/953#issuecomment-1310831008 might take another several MDs. So what I was supposed to do?

> instead of adding them to the tests

test is added in https://github.com/sebastianbergmann/php-code-coverage/blob/c4630fdec1def41d63ce7844d6e91ced5ccfa478/tests/_files/source_with_heavy_indentation.php#L95-L110

Reply:
```php
$var1 =
    'right'
    .
    $var1
;

$var1 =
    $var1
    .
    'left'
;
```
Where exactly are these two cases tested now in https://github.com/sebastianbergmann/php-code-coverage/pull/949 ?

Reply:
(as of PPH 8.0) the variable assignments are covered

the concat const expr does not produce a coverage line if the variable is const expr - this is tested a for ex. in l105 in the example I posted - I also tested locally concat is not different than any other binary operator (like plus) in sense of coverage